controllers/mattermost/mattermost/mattermost.go

```diff
@@ -381,7 +381,8 @@ func (r *MattermostReconciler) checkUpdateJob(
 				return nil, reconcileStatus{}, errors.Wrap(err, "Launching update image job failed")
 			}
 			recStatus.ResourcesReady = false
-			return nil, recStatus, errors.Wrap(err, "failed to restart update job")
+			reqLogger.Error(err, "failed to restart update job")
```

Comment:
Lets remove this error log, nothing actually failed here, we just did not find existing job so we started a new one.